benchmark/sqllogictest_tester/run_tester.sh

```diff
@@ -0,0 +1,304 @@
+    echo "$DOLT_GLOBAL_CONFIG" > "$DOLT_CONFIG_PATH"/config_global.json
+    dolt config --global --add user.creds "$CREDS_HASH"
+    dolt config --global --add metrics.disabled true
+    dolt version
+}
+
+function setup_testing_dir() {
+    rm -rf "$TMP_TESTING_DIR"
+    mkdir -p "$TMP_TESTING_DIR"
+
+    IFS=', ' read -r -a test_list <<< "$TEST_FILE_DIR_LIST"
+    for fd in "${test_list[@]}"
+    do
+         dir=$(dirname "$fd")
+         mkdir -p "$TMP_TESTING_DIR"/"$dir"
+         cp -r "$sqllogictest_checkout"/test/"$fd" "$TMP_TESTING_DIR"/"$fd"
+    done
+}
+
+function with_dolt_commit() {
+  local commit_hash="$1"
+    (
+      cd "$base_dir"/tempDolt/go
+      git checkout master
+
+      exists=$(git branch --list "temp-$commit_hash"| sed '/^\s*$/d' | wc -l)
+
+      if [ "$exists" -eq 0 ]; then
+        git checkout -b "temp-$commit_hash" "$commit_hash";
+      else
+        git checkout "temp-$commit_hash";
+      fi
+
+      git log -n 1
+
+      if ! [ -x "$base_dir"/.ci_bin/"$commit_hash"/dolt ]; then
+          if ! [ -d "$base_dir"/.ci_bin ]; then
+            mkdir -p "$base_dir"/.ci_bin/"$commit_hash"
+          fi
+
+          go get -mod=readonly ./...
+          go build -mod=readonly -o "$base_dir"/.ci_bin/"$commit_hash"/dolt ./cmd/dolt/.
+      fi
+    )
+
+    echo "Finished installing dolt from $commit_hash:"
+    export PATH="$base_dir/.ci_bin/$commit_hash":$old_path
+    dolt version
+}
+
+function with_dolt_checkout() {
+    (
+      cd "$base_dir"/go
+      if ! [ -x "$base_dir"/.ci_bin/checkout/dolt ]; then
+          if ! [ -d "$base_dir"/.ci_bin ]; then
+            mkdir -p "$base_dir"/.ci_bin/checkout
+          fi
+          go get -mod=readonly ./...
+          go build -mod=readonly -o "$base_dir"/.ci_bin/checkout/dolt ./cmd/dolt/.
+      fi
+    )
+    echo "Finished installing dolt from checkout:"
+    export PATH="$base_dir/.ci_bin/checkout":$old_path
+    dolt version
+}
+
+function import_once() {
+    local commit_hash="$1"
+    local test_num="$2"
+    local parsed="$log_dir/parsed-$commit_hash-$test_num".json
+
+    dolt checkout "$DOLT_BRANCH"
+
+    exists=$(dolt branch --list "temp-$commit_hash"| sed '/^\s*$/d' | wc -l | tr -d '[:space:]')
+
+    if [ "$exists" -eq 0 ]; then
+      dolt checkout -b "temp-$commit_hash";
+    else
+      dolt checkout "temp-$commit_hash";
+    fi
+
+    dolt table import -u nightly_dolt_results "$parsed"
+
+    dolt sql -r csv -q "\
+    select * from nightly_dolt_results;"\
+    > "$TMP_CSV_DIR"/"$commit_hash"_results.csv
+
+    dolt add nightly_dolt_results
+    dolt commit -m "add results for dolt at git commit $commit_hash ($test_num)"
+    dolt checkout master
+}
+
+#function import_parsed() {
+#    IFS=', ' read -r -a commit_list <<< "$COMMITS_TO_TEST"
+#    for c in "${commit_list[@]}"
+#    do
+#        seq 1 $TEST_N_TIMES | while read test_num; do
+#          import_once "$c" "$test_num"
+#        done
+#    done
+#}
```

Comment:
the commented out functions work, but are unused... fine to leave? or should I delete

Reply:
Probably fine to leave them around if you're going to need them shortly or something....